src/lib/pages/frontsite/PatronProfile/LoansList.js

```diff
@@ -1,37 +1,39 @@
-import { Pagination } from '@components/Pagination';
+import { RenderPagination } from './PaginationShow';
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
-import { Container, Item } from 'semantic-ui-react';
+import { Item } from 'semantic-ui-react';
 
 export default class LoansList extends Component {
+  renderPagination = () => {
+    const { activePage, isLoading, onPageChange, loans, rowsPerPage } =
+      this.props;
+
+    return (
+      <RenderPagination
+        currentPage={activePage}
+        currentSize={rowsPerPage}
+        loading={isLoading}
+        onPageChange={onPageChange}
+        items={loans}
+      />
+    );
+  };
+
   render() {
-    const {
-      activePage,
-      isLoading,
-      onPageChange,
-      loans,
-      rowsPerPage,
-      renderListEntry,
-      noLoansCmp,
-    } = this.props;
+    const { loans, renderListEntry, noLoansCmp, showAllLink } = this.props;
+
     return loans.total > 0 ? (
       <>
+        {loans.hits.length > 2 && this.renderPagination()}
```

Comment:
Why are you checking for more than `2`? BTW, you should use `loans.hits.total` instead of computing it (pay attention that it is different in ES6 and ES7, see examples in the code)